tools/tslint-rules/missingRollupExternalsRule.js

```diff
@@ -0,0 +1,41 @@
+const Lint = require('tslint');
+
+/**
+ * Rule that enforces that the specified external packages have been included in our Rollup config.
+ * Usage: [true, { config: './path/to/rollup/config.json', scope: ['^rxjs', '^@angular'] }]
+ */
+class Rule extends Lint.Rules.AbstractRule {
+  apply(file) {
+    return this.applyWithWalker(new Walker(file, this.getOptions()));
+  }
+}
+
+class Walker extends Lint.RuleWalker {
+  constructor(file, options) {
+    super(...arguments);
+
+    const args = options.ruleArguments[0];
+
+    if (!args.config) {
+      throw Error('missing-rollup-externals: The `config` argument has to be specified.');
+    }
+
+    this._scope = (args.scope || []).map(p => new RegExp(p));
+    this._configPath = args.config;
+    this._config = this._scope.length ? require(args.config) : {};
+  }
+
+  visitImportDeclaration(node) {
+    // Parse out the module name. The first and last characters are the quote marks.
+    const module = node.moduleSpecifier.getText().slice(1, -1);
+
+    // Check whether we match any of the scopes while missing the module from the config.
+    if (this._scope.some(p => p.test(module)) && !this._config[module]) {

```

Comment:
Yeah that doesn't really matter that much since it won't be really a public package.

As for convention I'd map it to something like `ng.material-examples`.